packages/adaptive-ui-figma-designer/src/core/serialization.ts

```diff
@@ -1,112 +1,45 @@
-import { StyleProperty } from "@adaptive-web/adaptive-ui";
-import { AdditionalData, AppliedDesignTokens, AppliedStyleModules, DesignTokenValues, PluginUINodeData } from "../core/model.js";
-
-/**
- * Serializable version of PluginUINodeData that works across Figma's iframe sandbox setup.
- */
-export interface SerializableNodeData {
-    id: string;
-    type: string;
-    name: string;
-    supports: Array<StyleProperty>;
-    children: SerializableNodeData[];
-    inheritedDesignTokens: string;
-    componentDesignTokens?: string;
-    designTokens: string;
-    componentAppliedStyleModules?: string;
-    appliedStyleModules: string;
-    componentAppliedDesignTokens?: string;
-    appliedDesignTokens: string;
-    additionalData: string;
-    effectiveAppliedStyleValues: string;
+export function mapReplacer(key: string, value: any): any {
+    if (value instanceof Map) {
+        return [["__dataType__", "Map"], ...value];
+        // Legacy format:
+        // return {
+        //     dataType: "Map",
+        //     value: [...value],
+        // };
```

Comment:
Can the comment be removed or is this being implemented at a later time?

Reply:
I left this to show the format that used to exist, since we still support deserialization of that format. I figured it didn't need to live very long since this is still beta and as far as I know we're the consumers of the tool now.

Reply:
It might not be clear that this code was moved from another file, so the comments are moving with it for context.